test/ApcTokenStorageTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace {  
```

Comment:
This was the only way I found to test with apc. Apc didn't work on cli, so I need to mock the apc_store and apc_fetch functions